crates/telio-relay/src/derp/mod.rs

```diff
@@ -250,10 +270,19 @@ impl State {
                     Ok(conn) => {
                         telio_log_info!("({}) Connected to {}", Self::NAME, server.get_address());
                         server.conn_state = RelayState::Connected;
+                        if let Some(aggregator) = aggregator.as_ref() {
```

Comment:
IMHO that's not a best place to place `aggregator`. I think we should sync this up with my [PR](https://github.com/NordSecurity/libtelio/pull/396). Because in this strategy there will be two aggregators (one for NAT traversal and one for DERP) and one instance can actually hold information for both cases. Maybe we need a channel, that goes down to `heartbeat` (Analytics) instance, and there the information gathering will take place ..?

Reply:
Nooo, aggregator shouldn't be private. This is just the right place to create it, just pass the shared pointer to Nurse.

Reply:
I think we need to sync up this part with @tomaszklak .

Reply:
I'm confused - aggregator is not constructed here, but in the `device/mod.rs` as part of the `Entieties` 🤔

Reply:
Oh, yeah, I saw `mod.rs` and I thought @matislovas talks about the creation inside `device/mod.rs` ^^'

Reply:
I was talking about storing `aggregator` inside `heartbeat.rs` Analytics struct.

Reply:
@matislovas Yeah, I got that, we probably don't want to do that